eval.py

```diff
@@ -25,10 +26,16 @@
 
     c = ExcelCompiler(file, ignore_sheets = ['IHS'])
     print "___Timing___ %s cells and %s named_ranges parsed in %s" % (str(len(c.cells)-len(c.named_ranges)), str(len(c.named_ranges)), str(datetime.now() - startTime))
-
     sp = c.gen_graph()
```

Comment:
if you are testing the evaluation of only one cell, you should generate the graph for only one output